<launch>
    <node pkg="magmed_joystick" type="readJoystick" name="readJoystick"/>
    <node pkg="magmed_feeder" type="magrobo_feeder" name="magrobo_feeder" output="screen"/>
    <node pkg="magmed_controller" type="joyCtrlFeed" name="joyCtrlFeed"/>
</launch>